run-test.sh

```diff
@@ -29,7 +29,7 @@ usage()
     echo "                                      default: <repo_root>/bin/Product/<OS>.x64.<ConfigurationGroup>"
```

Comment:
Yeah, discussed this in sync-up today. Corefx made changes to it involving the new test .builds files, we don't use it now but might we at some point? If we keep it I wouldn't expect to test it until such time as we needed it.